source/user/git_guide.rst

```diff
@@ -47,9 +52,10 @@ and keep it up-to-date with:
 
 .. code-block:: console
 
-    git fetch upstream
     git checkout develop
+    git fetch upstream
     git merge upstream/develop
+    git push origin develop
```

Comment:
I feel like this section conflicts with the branching section below. Assuming this guide is meant for contributors in general (otherwise users can just clone our repo directly), the more common usage would be the pattern below (checkout based on upstream, and rebase instead of merge). Would it make sense to remove this section, starting at `and keep it up-to-date with` until the `For Contributors`?

Reply:
I have improved the guide. Let me know what you think.